### DISCUSSION TITLE
Error: Could not configure server!

### DISCUSSION THREAD

**Discussion by Questv777** [vpnhood/VpnHood#160] [Q&A] [answered]
Hi Folks,

I downloaded the VpnHood Server

and ran it on windows server CMD, but it gave an error

**Comment by bobvhood** [comment, marked as answer]
You need to run it by a sudo user for 443 port. Use sudo.

**Reply by bobvhood** [reply]
It is ok; You need to be an Administrator to use the 443 port. If you have Windows Server, you need to do some work manually, such as disabling windows to listen 443 port, or you can use another port, such as 8443.

You can also use our access server, which makes the installation much easier.
https://github.com/vpnhood/VpnHood/wiki/VpnHood-Access-Server

Try to install it using the access server, and let us know if you still have a problem. In the last phase, you can share your Server username and password (if you like) so one of our experts will log in to your server and install it.

Good luck

**Reply by bobvhood** [reply]
It depends on your VPS configuration. I am not sure which process is listening to 443. You can send us (support@vpnhood.com) a temporary RDP credential to your windows so we can look at it.
